backend/helpers.py

```diff
@@ -23,18 +24,19 @@ def wrap_io(catch=Exception) -> (callable, callable):
         err.write(traceback.format_exc())
 
 
-def ask_for_temporary_mail():
+def ask_for_temporary_mail() -> Optional[dict]:
+    """
+    Return dict with connection info for ethereal.mail or None if failed.
+    """
```

Comment:
If I understand right this is the message in case etheral data does not load. If I'm right - it took me a while to figure it out. If I'm wrong - it took me a while not to figure it out anyway. Would it make sense to be a bit more descriptive here?

Reply:
Yes, in case ethereal dies/closes/stops working/api changes/whatever out of order happens - we return None which will indicate us that we failed to fetch credentials and we should turn off mail sending. I'll expand this docstring a bit.